client/src/components/ChatComponent/index.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable object-curly-newline */
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Layout, Icon, Avatar, Button, Input, Skeleton, List } from 'antd';
+
+const { Content } = Layout;
+const IconText = ({ type, text }) => (
+  <span>
+    <Icon type={type} style={{ marginRight: 8 }} />
+    {text}
+  </span>
+);
+IconText.propTypes = {
+  type: PropTypes.string.isRequired,
+  text: PropTypes.string.isRequired,
+};
+
+const listData = [];
+for (let i = 0; i < 3; i += 1) {
+  listData.push({
+    href: '#',
+    title: `username ${i}`,
+    avatar: 'https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png',
+    content:
+      'We supply a series of design principles, practical patterns and high quality design resources (Sketch and Axure), to help people create their product prototypes beautifully and efficiently.',
+  });
+}
+
+const ChatComponent = () => {
+  const [loading, setLoading] = useState(true);
+  const [messages] = useState(listData);
+  useEffect(() => {
+    setTimeout(() => setLoading(false), 2000);
+  }, []);
+  function sendMessage(e) {
+    console.log('sendingMessage', e.target.value);
+    e.target.value = '';
```

Comment:
Instead of directly manipulating the value here, try creating state for the message input and set the value to empty string on sending message.

Reply:
Removed the code for now.
Will update when integrating with sockets.